NitroxClient/Communication/Packets/Processors/CyclopsActivateHornProcessor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Packets;
+using System;
+using UnityEngine;
+
+namespace NitroxClient.Communication.Packets.Processors
+{
+    public class CyclopsActivateHornProcessor : ClientPacketProcessor<CyclopsActivateHorn>
+    {
+        private PacketSender packetSender;
+
+        public CyclopsActivateHornProcessor(PacketSender packetSender) {
+            this.packetSender = packetSender;
+        }
+
+        public override void Process(CyclopsActivateHorn packet) {
+            Optional<GameObject> opCyclops = GuidHelper.GetObjectFrom(packet.Guid);
+
+            if (opCyclops.IsPresent()) {
+                CyclopsHornControl horn = opCyclops.Get().GetComponentInChildren<CyclopsHornControl>();
+
+                if (horn != null) {
+                    using (packetSender.Suppress<CyclopsActivateHorn>()) {
+                        Utils.PlayEnvSound(horn.hornSound, horn.hornSound.gameObject.transform.position, 20f);
+                    }
```

Comment:
You are patching `OnPress`, which calls `PlayEnvSound`, so there's no need to suppress the packet unless you'd call `OnPress` directly here.

Reply:
Done in new push (https://github.com/Sunrunner37/Nitrox/pull/23/files)